gdnative-core/src/core_types/quat.rs

```diff
@@ -9,22 +12,248 @@ pub struct Quat {
     pub w: f32,
 }
 
+/// Helper methods for `Quat`.
+///
+/// See the official [`Godot documentation`](https://docs.godotengine.org/en/stable/classes/class_quat.html).
 impl Quat {
+    pub const IDENTITY: Self = Self::new(0.0, 0.0, 0.0, 1.0);
```

Comment:
maybe document this (stolen from official):

> The identity quaternion, representing no rotation. Equivalent to an identity [`Basis`] matrix. If a vector is transformed by an identity quaternion, it will not change.